maxent_cl.R

```diff
@@ -85,19 +98,22 @@ bio12_mutated<-calc(bio12, function(x) (x * precInc)+x)
 #create raster stack
 predictors<-stack(bio4, bio8, bio10_mutated, bio12_mutated, clay, sand)
 crs(predictors)<-"+proj=longlat +datum=WGS84 +no_defs +ellps=WGS84 +towgs84=0,0,0" 
-names(predictors)<-c("westAfrica_bio4", "westAfrica_bio8", "westAfrica_bio10", "westAfrica_bio12", "westAfrica_CLYPPT", "westAfrica_SNDPPT")
-
+if (modeltype == "swarm") {
+  names(predictors)<-c("westAfrica_bio_4", "westAfrica_bio_8", "westAfrica_bio_10", "westAfrica_bio_12", "westAfrica_clay", "westAfrica_sand")
```

Comment:
For some reason the names on the pre trained models differ slightly so I handled that.